elasticapm/instrumentation/register.py

```diff
@@ -37,6 +37,7 @@
     "elasticapm.instrumentation.packages.jinja2.Jinja2Instrumentation",
     "elasticapm.instrumentation.packages.psycopg2.Psycopg2Instrumentation",
     "elasticapm.instrumentation.packages.psycopg2.Psycopg2ExtensionsInstrumentation",
+    "elasticapm.instrumentation.packages.asyncio.asyncpg.AsyncPGInstrumentation",
```

Comment:
Please move this into the "asyncio" list starting at line 67.

Unfortunately, we haven't come up with a smarter way to avoid importing Python 3.5+ only code on 2.7. I guess a naive way would be to skip any instrumentations that have `.asyncio.` in the path. I'll investigate that.